en/sensor/sf1xx_lidar_setup.md

```diff
@@ -13,9 +13,10 @@ Driver supports only i2c connection.
 ## Configuring lidar
 --------------------------------------------------------
 
-You should connect to sensor via usb (it has internal usb to serial converter), run terminal, press `space` and check that i2c address equal to `0x66`.
+To check the i2c address of the Lidar, connect to the sensor to via usb (it has an internal serial to usb converter). Open up the device with a terminal emulator (such as minicom). You should see a stream of measurements in your terminal once you successfully connect. Press `space` to bring up the little menu. Look through that to find the I2C bus address. If it is not '0x66', change it to '0x66'.
 Newer sensor versions already have `0x66` preconfigured. Older have `0x55` which conflicts with `rgbled` module.
 
+Note: it is insufficient to simply hook up the lidar to the i2c port; you also need to provide external power via the microusb connection. So the final wiring should include the four wires connected to i2c, as well as a microusb cable providing external power. 
```

Comment:
That's strange because for me it was a direct correlation: if I unplugged external power, the driver would fail. If I plugged it back it, would work again. I was running the pixhawk on power from my laptop. I tried again today with 5V 2Amp adaptor and got the same result. I checked the lightware manual and it actually says you need to provide external power.
I'm using the SF11/c  
Page 5:
https://acroname.com/sites/default/files/assets/sf11_-_laser_altimeter_manual_-_rev_6.pdf